src/sentry/static/sentry/app/views/organizationContext.jsx

```diff
@@ -153,11 +166,23 @@ const OrganizationContext = createReactClass({
     fetchOrganizationDetails(
       this.props.api,
       this.getOrganizationSlug(),
-      this.props.detailed
+      this.props.detailed,
+      true // silent
```

Comment:
How do we update organization in organizationContext if this is silent?

Reply:
yup I just want to make sure not to reset the org store as to not lose the progress of the lightweight fetching

Reply:
I would add ^ message as the comment since it's more helpful to explain *why* you're doing it rather than what it is.